cmd/testpeers.go

```diff
@@ -38,19 +72,30 @@ func newTestPeersCmd(runFunc func(context.Context, io.Writer, testPeersConfig) e
 
 	bindTestFlags(cmd, &config.testConfig)
 	bindTestPeersFlags(cmd, &config)
+	bindP2PFlags(cmd, &config.P2P)
+	bindDataDirFlag(cmd.Flags(), &config.DataDir)
+	bindTestLogFlags(cmd.Flags(), &config.Log)
 
 	return cmd
 }
 
 func bindTestPeersFlags(cmd *cobra.Command, config *testPeersConfig) {
 	const enrs = "enrs"
-	cmd.Flags().StringSliceVar(&config.ENRs, "enrs", nil, "[REQUIRED] Comma-separated list of each peer ENR address.")
-	cmd.Flags().StringSliceVar(&config.P2PRelays, "p2p-relays", []string{"https://0.relay.obol.tech,https://2.relay.obol.tech"}, "Comma-separated list of each peer P2P relay address.")
+	cmd.Flags().StringSliceVar(&config.ENRs, enrs, nil, "[REQUIRED] Comma-separated list of each peer ENR address.")
```

Comment:
We already have a way to enforcing required flags at runtime (see what we're doing with the `exit` commands). No need to explicitly write "required".

Reply:
I also enforce them the same way afaict - `mustMarkFlagRequired`. This `[REQUIRED]` is only for UX, similar to the `create dkg` command.